ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Breakdown.kt

```diff
@@ -52,8 +52,12 @@ fun RBuilder.containerList(containers: List<FileContainer>, sizeType: Measurable
 @Suppress("UNUSED_PARAMETER")
 fun RBuilder.containerListItem(id: Int, container: FileContainer, sizeType: Measurable.SizeType, @RKey key: String) {
     div(classes = "accordion-item") {
-        containerListItemHeader(id, container, sizeType)
-        containerListItemBody(id, container, sizeType)
+        if (container.files.isEmpty()) {
+            containerWithoutFilesListItemHeader(container, sizeType)
+        } else {
+            containerListItemHeader(id, container, sizeType)
+            containerListItemBody(id, container, sizeType)
+        }
```

Comment:
UI change: if there isn't any list of files for a given component, it won't be shown as an expandable/collapsable row